book/src/running/virtual_machine/pci_passthrough.md

```diff
@@ -52,6 +52,13 @@ e.g. `echo 15b3 1019 > /sys/bus/pci/drivers/vfio-pci/new_id`
 
 Now, QEMU can be launched with direct access to the device.
 
+### Return device to the Host OS
+To reset the device, you can either reboot the system or return the device to the host OS using the following commands (replacing `$slot_info` with the value previously retrieved):  
+```sh
+echo 1 >/sys/bus/pci/devices/$slot_info/remove    
+echo 1 >/sys/bus/pci/rescan  
```

Comment:
to address this, you can either add "rescan" as a known word in our dictionary (or at the end of this file), or you can disabled checking for this code block. 

I assume there's a way to disable spell check for all code blocks, that setting should be enabled by default. Perhaps @apogeeoak knows how we could do that?

Reply:
I created a pull request #415 to disable spell check for code blocks.

Reply:
thanks so much!